pkg/branding/brandedlogo.go

```diff
@@ -0,0 +1,38 @@
+package branding
+
+import (
+	"embed"
+	"encoding/base64"
+
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+const (
+	rhacslogoFile    = "files/red_hat_acs_logo_rgb.png"
+	stackroxLogoFile = "files/stackrox_logo_wide_dkblue.png"
+)
+
+var (
+	//go:embed files/red_hat_acs_logo_rgb.png
+	logoRHACS       embed.FS
+	logoRHACSBase64 = func() string {
+		bytes, err := logoRHACS.ReadFile(rhacslogoFile)
+		utils.Must(err)
+		return base64.StdEncoding.EncodeToString(bytes)
+	}()
+	//go:embed files/stackrox_logo_wide_dkblue.png
+	logoStackRox       embed.FS
```

Comment:
It looks like you can use wildcards in `// go:embed` directive.
https://github.com/stackrox/stackrox/blob/5a8181db4e98c0ac838e6d8f8b93e30b9ff5e039/pkg/defaults/policies/policies.go#L24-L25
In this case you don't need to define two instances of `embed.FS`: `logoRHACS` and `logoStackRox`. It will be sufficient to do just one and read both files from it.

Reply:
Ok